charts/jenkins/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Use the following links to reference issues, PRs, and commits prior to v2.6.0.
 The changelog until v1.5.7 was auto-generated based on git commits.
 Those entries include a reference to the git commit to be able to get more details.
 
+## 5.0.19
```

Comment:
I think this should be a minor version bump as it would be new functionality

Reply:
Makes sense - updated to 5.1.0